tox.ini

```diff
@@ -0,0 +1,13 @@
+# Tox (http://tox.testrun.org/) is a tool for running tests
+# in multiple virtualenvs. This configuration file will run the
+# test suite on all supported python versions. To use it,
+# "pip install tox" and then run "tox" from this directory.
+
+[tox]
+envlist = py27, py33, py34, py35, pypy, pypy3
+skip_missing_interpreters = True
+
+[testenv]
+commands =
+    {envpython} setup.py test
+
```

Comment:
Can you add `deps = pillow`. This will run the test suite for `ImageField` https://github.com/mongodb/pymodm/blob/0.4.0/test/field_types/test_image_field.py#L23-L32.